src/i18next.translate.js

```diff
@@ -134,6 +134,10 @@ function _translate(potentialKeys, options) {
         options = options || {};
     }
 
+    if (typeof o.defaultVariables == 'object') {
+        options = f.extend({}, o.defaultVariables, options);
```

Comment:
This does not allow you to have default nested values. i.e. 

```
defaultVariables: {
      foo: { bar:  'something' },
    },
```

Reply:
PR welcome for any typescript related fixes / improvements